deploy/charts/cert-manager/values.yaml

```
@@ -421,7 +421,9 @@ enableServiceLinks: false
 prometheus:
   # Enable Prometheus monitoring for the cert-manager controller to use with the
   # Prometheus Operator. If this option is enabled without enabling `prometheus.servicemonitor.enabled` or
-  # `prometheus.podmonitor.enabled`, 'prometheus.io' annotations are added to the cert-manager Deployment resources.
+  # `prometheus.podmonitor.enabled`, 'prometheus.io' annotations are added to the cert-manager Deployment
+  # resources. Additionally, for backwards compatibility a service is created which can be used together
+  # with your own ServiceMonitor (managed outside of this Helm chart).
```

```suggestion
  # `prometheus.podmonitor.enabled`, 'prometheus.io' annotations are added to the cert-manager Deployment
  # resources. Additionally, a service is created which can be used together
  # with your own ServiceMonitor (managed outside of this Helm chart).
```

It's got nothing to do with backwards compatibility. 
Some users *will* want to bring their own ServiceMonitor resource or simply want to use the Service with some other monitoring system.